From e5e71f0d4ff102af225fb8cf2ab0e53215ea242f Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Fri, 29 Sep 2017 18:21:23 +0800
Subject: [PATCH 92/98] dt-binding: add display pipeline for U-Boot Allwinner
 R40 simplefb

The SimpleFB of the U-Boot on R40 currently uses R40's mixer0, TCON_TV0
and HDMI controller.

Add display pipeline string for it.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 Documentation/devicetree/bindings/display/simple-framebuffer-sunxi.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/display/simple-framebuffer-sunxi.txt b/Documentation/devicetree/bindings/display/simple-framebuffer-sunxi.txt
index 29b553933e46..019e1e5f6000 100644
--- a/Documentation/devicetree/bindings/display/simple-framebuffer-sunxi.txt
+++ b/Documentation/devicetree/bindings/display/simple-framebuffer-sunxi.txt
@@ -19,6 +19,7 @@ Required properties:
   "mixer0-lcd0-hdmi"
   "mixer1-lcd1-hdmi"
   "mixer1-lcd1-tve"
+  "mixer0-tcon_tv0-hdmi"
 
 Example:
 
-- 
2.13.6

